tests/js/babel-plugins/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,1529 @@
+  |> (x) => doubleSay(x, ", ")
+  |> capitalize
+  |> (x) => x + "!"
+  |> (x) => new User.Message(x)
+  |> (x) => stream.write(x)
+  |> await
+  |> console.log;
+
+const result = exclaim(capitalize(doubleSay("hello")));
+result; //=> "Hello, hello!"
+
+const result = "hello" |> doubleSay |> capitalize |> exclaim;
+
+result; //=> "Hello, hello!"
+
+const person = { score: 25 };
+
+const newScore =
+  person.score |> double |> (n) => add(7, n) |> (n) => boundScore(0, 100, n);
+
+newScore; //=> 57
+
+// As opposed to:
+let newScore = boundScore(0, 100, add(7, double(person.score)));
+
+================================================================================
+`;
+
+exports[`pipeline-operator-minimal.js format 1`] = `
+====================================options=====================================
+parsers: ["babel", "babel-ts", "babel-flow"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+// https://babeljs.io/docs/en/babel-plugin-proposal-pipeline-operator
+// https://github.com/tc39/proposal-pipeline-operator/
+
+let result = exclaim(capitalize(doubleSay("hello")));
+result //=> "Hello, hello!"
+
+let result = "hello"
+  |> doubleSay
+  |> capitalize
+  |> exclaim;
+
+result //=> "Hello, hello!"
+
+=====================================output=====================================
+// https://babeljs.io/docs/en/babel-plugin-proposal-pipeline-operator
+// https://github.com/tc39/proposal-pipeline-operator/
+
+let result = exclaim(capitalize(doubleSay("hello")));
+result; //=> "Hello, hello!"
+
+let result = "hello" |> doubleSay |> capitalize |> exclaim;
+
+result; //=> "Hello, hello!"
+
+================================================================================
+`;
+
+exports[`pipeline-operator-smart.js format 1`] = `
+====================================options=====================================
+parsers: ["babel", "babel-ts", "babel-flow"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+// https://babeljs.io/docs/en/babel-plugin-proposal-pipeline-operator
+// https://github.com/js-choi/proposal-smart-pipelines
+
+value
+|> await #
+|> doubleSay(#, ', ')
+|> capitalize // This is a function call.
+|> # + '!'
+|> new User.Message(#)
+|> await #
+|> console.log; // This is a method call.
+
+// (The # token isn't final; it might instead be @ or ? or %.)
+
+=====================================output=====================================
+// https://babeljs.io/docs/en/babel-plugin-proposal-pipeline-operator
+// https://github.com/js-choi/proposal-smart-pipelines
+
+value
+|> await #
+|> doubleSay(#, ", ")
+|> capitalize  // This is a function call.
+  |> # + "!"
+  |> new User.Message(#)
+  |> await #
+  |> console.log; // This is a method call.
```

Comment:
Why this pipeline format like this? Bug?

Reply:
@sosukesuzuki

Reply:
Which is the input file for the output?

Reply:
Shouldn't it  be  

```js
value
  |> await #
  |> doubleSay(#, ", ")
  |> capitalize  // This is a function call.
  |> # + "!"
  |> new User.Message(#)
  |> await #
  |> console.log; // This is a method call.
```

All aligned?

Reply:
Seems happens in  `&&` too, I'll open an issue later.

Reply:
I see. Thank you.

Reply:
#8450

Reply:
probably you mistook the issue number:smile: